### PR TITLE
fixed incorrect link

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2307

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2012-ps/internationalcmdlets/Get-WinUserLanguageList.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ The generic list object supports the following methods:
 
 The output language object contains the following properties:
 
-- BCP-47 (READ). A standard language tag that is used to identify languages. For more information, see the Internet Engineering Task Force (IETF) BCP 47 RFChttp://go.microsoft.com/fwlink/?LinkID=242207. 
+- BCP-47 (READ). A standard language tag that is used to identify languages. For more information, see the Internet Engineering Task Force (IETF) [BCP 47 RFC](http://go.microsoft.com/fwlink/?LinkID=242207).
```
Comment (dariomws) [inline docset/winserver2012-ps/internationalcmdlets/Get-WinUserLanguageList.md:103]:
```suggestion
- BCP-47 (READ). A standard language tag that is used to identify languages. For more information, see the [Internet Engineering Task Force (IETF) BCP 47 RFC](https://go.microsoft.com/fwlink/?LinkID=242207). 
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by get-itips — APPROVED** [review]
I am ok once Dario suggestion is applied.

**Review by tseward — APPROVED** [review]
Also good after the suggestion is applied.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**